src/client.js

```diff
@@ -611,6 +674,10 @@ MatrixClient.prototype.initCrypto = async function() {
         "crypto.roomKeyRequest",
         "crypto.roomKeyRequestCancellation",
         "crypto.warning",
+        "crypto.devicesUpdated",
+        "deviceVerificationChanged",
+        "userVerificationChanged",
+        "cross-signing.keysChanged",
```

Comment:
Yeah, I named it `cross-signing` because we're using it as a hyphenated word rather than as two words, but I'm fine with changing it to camel-case if that's how we want to do things.